bitrise.yml

```diff
@@ -102,13 +102,13 @@ workflows:
             "$BITRISE_SOURCE_DIR/firefox-ios/Client/" \
             "$BITRISE_SOURCE_DIR/xcodebuild.log"
     - deploy-to-bitrise-io@2:
-        run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests"}}'
+        run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests" | and (getenv "BITRISE_XCRESULT_PATH" | ne "")}}'
```

Comment:
When tests run the BITRISE_XCRESULT_PATH env variable points to the test result file

Reply:
thanks for clarifying! do you have the original PR that was merged in which the deploy step would fail when unit tests were failing?

Reply:
and this is one example of the confusing error I was talking about: https://app.bitrise.io/build/7a996b63-cdf1-4930-9f46-9e1160123d13?tab=log